crypto/fipsmodule/sha/sha3_test.cc

```diff
@@ -22,10 +22,8 @@ class SHA3TestVector {
 
   bool ReadFromFileTest(FileTest *t);
   
-  void NISTTestVectors() const {
-    uint32_t digest_length = SHA3_256_DIGEST_LENGTH;
-    const EVP_MD* algorithm = EVP_sha3_256();
-    uint8_t digest[SHA3_256_DIGEST_LENGTH];
+  void NISTTestVectors(const EVP_MD *algorithm, uint8_t *digest) const {
```

Comment:
Any reason you pull out `digest` as a parameter? C++ can do other variable-length heap allocations.

Reply:
I thought it may be more "clean" way to do that. I agree about the heap, just changed it. Thanks!